packages/core/http/core-http-route-validator-internal/index.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export { RouteValidator } from './src/validator';
```

Comment:
Do we really need another package just for the validator?

Reply:
This enabled us to use this same validation across the router and versioned router packages (otherwise we'd have a circular dep). Collapsing them into one removes this 🙌🏻